Examples/Swift/ViewController.swift

```diff
@@ -254,37 +262,6 @@ extension ViewController: NavigationViewControllerDelegate {
 
 extension ViewController: WaypointConfirmationViewControllerDelegate {
     func confirmationControllerDidConfirm(controller confirmationController: WaypointConfirmationViewController) {
```

Comment:
Per https://github.com/mapbox/mapbox-navigation-ios/pull/270#discussion_r126295971, rename `controller` to `_`.